framework/auth/oauth_scopes.py

```diff
@@ -38,6 +38,12 @@ class CoreScopes(object):
     NODE_REGISTRATIONS_READ = 'nodes.registrations_read'
     NODE_REGISTRATIONS_WRITE = 'nodes.registrations_write'
 
+    NODE_COMMENTS_READ = 'comments_read'
```

Comment:
For the string values of these named constants, try to follow this rough naming convention: `collection.subcollection_permission`

So for example: `comments.data_read` and `comments.reports_read` might be good values.